tips.json

```diff
@@ -52,6 +52,18 @@
       "tip": "git remote set-url origin <URL>"
   },
   {
+    "title": "Get list of all remote references",
+    "tip": "git remote || git remote show"
```

Comment:
i wanted to show either of the two can be used, hence used this. Can you suggest something else?